package.json

```diff
@@ -19,7 +19,10 @@
   "scripts": {
     "test": "make test"
   },
-  "dependencies": {},
+  "dependencies": {
+    "async": "^2.1.5",
```

Comment:
Because `async` is only used by the build script it should only be included in `devDependencies`.